tests/xdg_surface_stable.cpp

```diff
@@ -129,10 +129,9 @@ TEST_F(XdgSurfaceStableTest, creating_xdg_surface_from_wl_surface_with_committed
 {
     wlcs::Client client{the_server()};
 
-    auto* const xdg_wm_base = static_cast<struct xdg_wm_base*>(
-        client.acquire_interface("xdg_wm_base", &xdg_wm_base_interface, 1));
+    auto const xdg_wm_base = client.bind_if_supported<struct xdg_wm_base>(AnyVersion);
 
-    auto* const surface = wl_compositor_create_surface(client.compositor());
+    wlcs::WlHandle<wl_surface> const surface{wl_compositor_create_surface(client.compositor())};
```

Comment:
Not really about this PR, but this looks ugly. Maybe we should have
```c++
auto const surface = wlcs::make_handle(wl_compositor_create_surface(client.compositor()));
```

Reply:
We already have `wlcs::wrap_wl_object`; I've switched these lines to using it.